### PR TITLE
Slightly customize the home page for users

### PR DESCRIPTION
**Pull request by edpaget**
Returning users will now see the four projects they most recently
classified on and the four featured projects instead of seeing the
explanation of the Zooniverse like they do now.

screen shot 2015-09-08 at 2 19 48 pm

### CODE REVIEW

**Comment by edpaget** [comment]
tagging @DarrenMcRoy @heathv for input.

**Comment by DarrenMcRoy** [comment]
Needs some type of punctuation at the end there. Colon? Ellipsis? Question mark?

I vote for colon. "... recent projects:"

**Comment by edpaget** [comment]
Want to deploy it and see what people think?

**Comment by camallen** [comment]
i think we should

**Comment by chrissnyder** [comment]
Yea will look at it soon as im out of this training thing

File: app/pages/home.cjsx [hunk 1]
```diff
@@ -41,46 +64,64 @@ module.exports = React.createClass
...
-        <div className="about-items-list">
-          {for item in aboutItems
-            <div key={item} className="about-item">
-              <div className="about-item-wrapper">
-                <img className="about-image" src="./assets/home-#{item}.gif" alt="" />
-                <div className="about-item-content">
-                  <Translate component="h5" content="home.about.#{item}.title" />
-                  <Markdown>{counterpart "home.about.#{item}.content"}</Markdown>
-                </div>
+        {if @props.user
```
Comment (chrissnyder) [inline app/pages/home.cjsx]: Could this be dependent upon if both a user exists and they've actually classified on something?

File: app/pages/home.cjsx [hunk 2]
```diff
@@ -30,7 +31,29 @@ counterpart.registerTranslations 'en',
         content: '''Our platform offers many opportunities for education, from using projects in classrooms to sharing information between volunteers. You can even use the [Zooniverse Project Builder](/lab) to create your very own project!'''
     featuredProjects:
       title: 'Get started on a project right now!'
+      loggedTitle: 'Get started on a new project right now!'
```
Comment (chrissnyder) [inline app/pages/home.cjsx]: On purpose these two are the same?

Comment (edpaget) [inline app/pages/home.cjsx]: They're slightly different (new vs not new)

Comment (chrissnyder) [inline app/pages/home.cjsx]: ... don't I feel stupid.

File: app/pages/home.cjsx (hunk 1 above)
Comment (edpaget) [inline app/pages/home.cjsx]: Done. Now it loads 4 projects from the list.

**Comment by chrissnyder** [comment]
Really nice! Could you remove the log at https://github.com/zooniverse/Panoptes-Front-End/pull/1539/files#diff-4e943d3edc8de512994b6a0031c1ed97R80? Good to go otherwise

**Pull request merged**